pwd_wrapper.py

```diff
@@ -55,7 +55,7 @@ def validate_args(args):
         raise FileNotFoundError("Couldn't find the pwd_guess.py script at {}".format(pwd_guess))
     statvfs = os.statvfs(args.rundir)
     free_bytes = statvfs.f_frsize * statvfs.f_bfree
-    if free_bytes < 500000000:
+    if free_bytes < 200000000:
```

Comment:
What's the reason for this constant. Does it really sometimes take more than 200000000 bytes? Also, could you specify this as a constant at the beginning of the file, and document the size in human readable terms (e.g., 2GB)?

Reply:
The model size is difficult to encode. This is just for a basic sanity check to ensure that the disk space is not full. The problem I faced was that the program would crash after a few hours of processing. This line just gives an error to prevent the crash happening down the road. 
I don't know how to represent the 2GB in human readable forms and still do the comparison